### PR TITLE
docs(events): add description about events

### PR DESCRIPTION
**Pull request by davidsonfellipe**
Add a description about regularly events and meetings

Screen Shot 2021-09-09 at 8 11 13 AM

### CODE REVIEW

**Review by gillianomenezes — APPROVED** [review]
awesome! :rocket:

File: events/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+### [Events at Loadsmart]
```
Comment (barbosa) [inline events/README.md:1]:
```suggestion
### Events at Loadsmart
```

File: events/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+Documentation on regularly occurring events and meetings
+
+<!-- prettier-ignore-start -->
+<!-- start_toc -->
+| Doc | Overview |
+|--|--|
+| [Tech Talks](/events/tech-talks.md#readme) | How tech talks work on loadsmart |
```
Comment (barbosa) [inline events/README.md:9]:
```suggestion
| [Tech Talks](/events/tech-talks.md#readme) | How tech talks work at Loadsmart |
```

File: events/tech-talks.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+title: Tech Talks
+description: How tech talks work on loadsmart
+---
+
+Everyone is welcomed and encouraged to submit a talk! Whether you have been working as a senior engineer or intern, experienced or 1st-time speaker, we want to hear what you are excited about
+
+Even if we already have a tech talk about a given subject - submit it anyway! We will love to hear your perspective on the subject.
+
+Would you like to present something at the next Loadsmart Tech talk? Send a Slack message in #tech-talks 🔒 channel.
```
Comment (barbosa) [inline events/tech-talks.md:10]:
```suggestion
Would you like to present something at the next Loadsmart Tech talk? Send a Slack message to #tech-talks 🔒 channel.
```

**Review by barbosa — CHANGES_REQUESTED** [review]
We might want to consider having some of the presentations open as well 💡

**Pull request merged**